2020-11-30 12:42:43,210--test_register.py--70--INFO--hanjiping_log--不输入密码该条用例通过
2020-11-30 12:42:43,851--test_register.py--66--ERROR--hanjiping_log--注册名11位该条测试用例未通过
2020-11-30 12:42:43,851--test_register.py--67--ERROR--hanjiping_log--2 != 0 within 7 places (2 difference)
Traceback (most recent call last):
  File "/Users/hanjiping/PycharmProjects/Port_test01/TestCase/test_register.py", line 49, in test_register
    self.assertAlmostEqual(expected['code'],res['code'])
  File "/opt/anaconda3/lib/python3.7/unittest/case.py", line 906, in assertAlmostEqual
    raise self.failureException(msg)
AssertionError: 2 != 0 within 7 places (2 difference)
2020-11-30 12:42:44,246--test_register.py--70--INFO--hanjiping_log--类型输入非0，1内容该条用例通过
2020-11-30 12:42:44,862--test_register.py--66--ERROR--hanjiping_log--手机号为字符串类型该条测试用例未通过
2020-11-30 12:42:44,862--test_register.py--67--ERROR--hanjiping_log--2 != 0 within 7 places (2 difference)
Traceback (most recent call last):
  File "/Users/hanjiping/PycharmProjects/Port_test01/TestCase/test_register.py", line 49, in test_register
    self.assertAlmostEqual(expected['code'],res['code'])
  File "/opt/anaconda3/lib/python3.7/unittest/case.py", line 906, in assertAlmostEqual
    raise self.failureException(msg)
AssertionError: 2 != 0 within 7 places (2 difference)
